lib/internal/test_runner/utils.js

```diff
@@ -74,10 +77,21 @@ function convertStringToRegExp(str, name) {
   }
 }
 
+let _module;
+function setupTestReporters(reporter) {
+  if (getOptionValue('--test-reporter')) {
+    _module ??= new Module('node:test');
+    const reporters = _module.require(getOptionValue('--test-reporter'));
+    return compose(reporter, reporters);
```

Comment:
If we are going to support multiple reporters is the `reporter_stream` expected to do things like handle backpressure? Also, I'm assuming we won't want to `compose()` all of them together.

Reply:
We have two options:
1. either a reporter will have the same signature as `TapStream` and `ReporterStrem` (i.e implement all methods `ok`, `fail`, `plan`, `subtest` etc - it doesn't even have to inherit from `stream.Readable`):
this way each reporter can also be in charge of things like its destination
2. use the approach in this PR - and build all reporters as a duplex stream on top of `ReporterStrem` 


both of the options have pros and cons

Reply:
I would make `ReporterStream` broadcast events to all of the reporters and let them deal with buffering, backpressure, and other stream related concerns.

Reply:
When you say broadcast do you mean stream?

Reply:
I guess that's something we should experiment with a bit. Personally, I try to avoid streams when possible because of their complexity and speed. When I initially wrote `TapStream`, I intentionally made that the only part of `node:test` that was streaming because it had to deal with stdout - but I input data into the stream via the TAP API functions (`version()`, `plan()`, etc.). A non-stream `EventEmitter` might be adequate for sending data to the various reporters, but it's hard to say without some experimentation.